cognee/shared/utils.py

```diff
@@ -28,11 +48,14 @@ def send_telemetry(event_name: str, user_id, additional_properties: dict = {}):
     current_time = datetime.datetime.now()
     properties = {
         "time": current_time.strftime("%m/%d/%Y"),
+        "user_id": user_id,
         **additional_properties,
     }
 
+    posthog.identify(get_anonymous_id(), properties)
+
     try:
-        posthog.capture(user_id, event_name, properties)
+        posthog.capture(get_anonymous_id(), event_name, properties)
```

Comment:
Do we want to identify users by their unique ID? Or have it as a property so we can filter out same users